### DISCUSSION TITLE
ModuleNotFoundError: No module named 'pymonctl'

### DISCUSSION THREAD

**Discussion by Danie10** [tryptech/obs-zoom-and-follow#129] [Q&A] [answered]
Error in script log after starting OBS on Manjaro KDE with Python 3.10.11. This is after I had gone and done force reinstalls of PyMonCtl and PyWinCtl as I saw they had been updated in the last month or so. I did download this projects file and first ran the requirements script, which gave no errors.

The thing is PyWinCtl is definitely installed. I even ran it's own test script to make sure it actually functions. It does all the changes of monitor brightness, orientation, resolution, etc. So it is installed and visible to my Python on CLI. The OBS Zoom and Follow script has loaded and worked before on my computer. I just deleted it when I now downloaded the latest version to test again.

Question: It fails loading PyMonCtl on startup, but that is clearly installed and working under my Python. Is there anything else I could check why this script fails to import it? I may be missing something super obvious (it used to work before).

**Comment by tryptech** [comment, marked as answer]
Off the top of my head, if any dependency fails to load then it usually isn't installed under the version of Python that OBS is referencing. Double check that the default Python version/install on your system lines up with the Python referenced in OBS Python Settings. It may be the case that it references a different Python install that does not have the dependencies installed

**Comment by Danie10** [comment]
Yep I think you hit the nail on the head there.  It may be related to the issue at https://github.com/obsproject/obs-studio/issues/5628 in that OBS is seeing something different from the global Python the OS is using. I research that further.
